Portal/src/Datahub.Portal/Pages/Workspace/WebApp/WorkspaceWebAppPage.razor

```diff
@@ -0,0 +1,186 @@
+﻿@using Datahub.Application.Services.Security
+@using Datahub.Portal.Views.Dialogs
+@using Datahub.Shared.Entities
+@using Microsoft.Identity.Web
+@using System.Text.Json.Nodes
+@using Datahub.Infrastructure.Queues.Messages
+@using MediatR
+
+@inject IDbContextFactory<DatahubProjectDBContext> _dbContextFactory
+@inject IDialogService _dialogService
+@inject ISnackbar _snackbar
+@inject ILogger<WorkspaceWebAppPage> _logger
+@inject IMediator _mediatr
+
+<MudStack>
+    <MudStack Row="true">
+        <MudText Typo="Typo.h1">
+            @Localizer["Web Application"]
+        </MudText>
+        <MudSpacer/>
+        @if (_isProvisioned)
+        {
+            <MudButton Color="Color.Primary" Size="Size.Medium" Variant="Variant.Filled" OnClick="async () => { await ConfigureWebApp(); }">
+                @Localizer["Configure "]
+                <MudIcon Icon="@Icons.Material.Filled.Settings"/>
+            </MudButton>
+        }
+    </MudStack>
+
+    <MudText>
+        @Localizer["Here you can manage your web application. You can view and edit the configurations of your web application"]
+    </MudText>
+
+    <MudText Typo="Typo.h2">
+        @Localizer["Web Application Info"]
+    </MudText>
+    @if (_isProvisioned)
+    {
+        <MudText>
+            @Localizer["Host: "] <MudLink Href="@_webAppHost">@_webAppHost</MudLink>
+        </MudText>
+        <MudText>
+            @Localizer["Id: "] <code>@_webAppId</code>
+        </MudText>
+        <MudText>
+            @Localizer["Status: "]
+        </MudText>
+        @if (_isConfigured)
+        {
+            <MudText>
+                @Localizer["Framework: "] <code>@_webAppFramework</code>
+            </MudText>
+            <MudText>
+                @Localizer["Git repo: "] <code>@_webAppRepo</code>
+            </MudText>
+            <MudText>
+                @Localizer["Compose path: "] <code>@_webAppCompose</code>
+            </MudText>
+        }
```

Comment:
Nice, check out the updated simple table format on the database page, makes it all nice and clean